merlin/table/conversions.py

```diff
@@ -131,8 +185,7 @@ def _tensor_table_from_pandas_df(df: pd.DataFrame):
 
 @tensor_table_from_df.register_lazy("cudf")
 def _register_tensor_table_from_cudf_df():
-    import cudf
-
+    from merlin.core.compat import cudf
```

Comment:
This import forced me, via interrogate, to add in comments for all public methods in the file.

Reply:
could changing this cause unexpected runtime errors later in the case of cudf being installed but no visible GPUs available?